packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -2011,7 +2006,7 @@ function handleThrow(root: FiberRoot, thrownValue: any): void {
       // TODO: Suspending the work loop during the render phase is
       // currently not compatible with sibling prerendering. We will add
       // this optimization back in a later step.
-      !enableSiblingPrerendering &&
+      false &&
```

Comment:
?

Reply:
Removed this logic since its now dead code. Left some of the high level comments for when we get to reimplementing.